alpine/rc.conf

```diff
@@ -0,0 +1,241 @@
+# Global OpenRC configuration settings
```

Comment:
I think a full configuration file will help our users when they want to override our settings but it makes sense if this file contains only the setting we set. Will update and add a ref to default one.